From 8aeb94e772bc6d092606e7e2b0e75c072ba3b3c6 Mon Sep 17 00:00:00 2001
From: Luwei Zhou <b45643@freescale.com>
Date: Mon, 14 Oct 2013 15:56:41 +0800
Subject: [PATCH 423/449] ENGR00283520 linux-test: mxc_mlb_test: Enable MLB
 test for IMX6Q

Remove the IMX6Q out of excluding list. Fix several warning when compiling

Signed-off-by: Luwei Zhou <b45643@freescale.com>
---
 test/mxc_mlb_test/Makefile          |  2 +-
 test/mxc_mlb_test/mxc_mlb150_test.c | 14 +-------------
 2 files changed, 2 insertions(+), 14 deletions(-)

diff --git a/test/mxc_mlb_test/Makefile b/test/mxc_mlb_test/Makefile
index 553a823..10cb17a 100644
--- a/test/mxc_mlb_test/Makefile
+++ b/test/mxc_mlb_test/Makefile
@@ -2,7 +2,7 @@
 # Makefile for MediaLB driver test app.
 #
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX25_3STACK IMX6Q IMX6S
+EXCLUDE_LIST:= IMX21ADS IMX27ADS IMX31ADS IMX32ADS IMX37_3STACK IMX25_3STACK IMX6S
 
 INC += -I$(LINUXPATH)/include
 
diff --git a/test/mxc_mlb_test/mxc_mlb150_test.c b/test/mxc_mlb_test/mxc_mlb150_test.c
index 847b1c2..43c7374 100644
--- a/test/mxc_mlb_test/mxc_mlb150_test.c
+++ b/test/mxc_mlb_test/mxc_mlb150_test.c
@@ -48,16 +48,6 @@ static int t_case = 0;
 static unsigned int sync_quad = 0;
 static unsigned int isoc_pkg_len = 0;
 
-static unsigned int fps_set[] = {
-	256,
-	512,
-	1024,
-	2048,
-	3072,
-	4096,
-	6144,
-};
-
 int do_basic_test(int fd);
 int do_txrx_test(int fd);
 
@@ -74,7 +64,6 @@ void print_help(void)
 int main(int argc, char *argv[])
 {
 	int ret, flags;
-	int fd_set[4] = { 0 };
 	char test_case_str[10] = { 0 };
 
 	while (1) {
@@ -188,9 +177,8 @@ int main(int argc, char *argv[])
 
 int do_basic_test(int fd)
 {
-	int i, ret;
+	int ret;
 	unsigned long ver;
-	unsigned int tfps;
 	unsigned char addr = 0xC0;
 
 	/* ioctl check */
-- 
1.8.0

